crates/proposal-signing-backends/src/dkg.rs

```diff
@@ -96,73 +106,38 @@ impl super::ProposalSigningBackend for DkgProposalSigningBackend {
         tracing::debug!(
             nonce = %hex::encode(nonce.encode()),
             resource_id = %hex::encode(resource_id.into_bytes()),
-            src_chain_id = ?src_chain_id,
+            src_chain_id = ?self.typed_chain_id,
             proposal = %hex::encode(proposal.to_vec()),
             "sending proposal to DKG runtime"
         );
+
         let xt = tx_api.acknowledge_proposal(
             nonce,
             src_chain_id,
             ResourceId(resource_id.into_bytes()),
             BoundedVec(proposal.to_vec()),
         );
+        // webb dynamic payload
+        let execute_proposal_tx = WebbDynamicTxPayload {
+            pallet_name: Cow::Borrowed("dkgProposals"),
+            call_name: Cow::Borrowed("acknowledge_proposal"),
+            fields: vec![
+                Value::u128(u128::from(self.typed_chain_id.chain_id())),
+                Value::from_bytes(xt.call_data().encode()),
+            ],
+        };
```

Comment:
This call is wrong, first of all the pallet name is `DKGProposals` as you can find it here in the runtime:
https://docs.rs/webb/0.5.21/src/webb/substrate/dkg_runtime.rs.html#14

Another this is the parameters does not look right to me, this call takes 4 args (nonce, src_chain_id, resource_id, proposal_bytes)
Here is a suggested fix:
```suggestion
        let execute_proposal_tx = WebbDynamicTxPayload {
            pallet_name: Cow::Borrowed("DKGProposals"),
            call_name: Cow::Borrowed("acknowledge_proposal"),
            fields: vec![
                Value::u128(u128::from(nonce)),
                Value::u128(u128::from(self.typed_chain_id.chain_id())),
                Value::from_bytes(ResourceId(resource_id.into_bytes()).encode()),
                Value::from_bytes(BoundedVec(proposal.to_vec()).encode()),
            ],
        };
```